inspirehep/modules/migrator/tasks.py

```diff
@@ -187,17 +194,15 @@ def add_citation_counts():
             es,
             query={
                 "_source": "references.recid",
-                "filter": {
-                    "exists": {
-                        "field": "references.recid"
-                    }
-                },
                 "size": CHUNK_SIZE
             },
             scroll=u'1m',
             index="hep",
             doc_type="record"):
 
+        # initialize the key for record in citations_lookup dict in case it doesn't exist there yet
+        citations_lookup[record['_id']] = citations_lookup.get(record['_id'], 0)
```

Comment:
Here's the misunderstanding. it should not be `citations_lookup[record['_id']]`, rather:
```python 
for recid in record['references.recid']:
    citations_lookup[recid]
```